src/hooks/posters.tsx

```diff
@@ -10,6 +16,7 @@ import { tokeniseStringWithQuotesBySpaces } from "utils/text";
 import { isLoaded, useFirestoreConnect } from "./useFirestoreConnect";
 import { useSelector } from "./useSelector";
 import { useDebounceSearch } from "./useDebounceSearch";
+import { WithId } from "../utils/id";
```

Comment:
Please ensure imports use their canonical paths, not `../` relative paths.

This should also be sorted with the other `utils/*`
```suggestion
import { WithId } from "utils/id";
```

Reply:
IDE artefact I had already dealt with. Took a while to catch it.

Reply:
done